pkg/discovery/monitoring/kubelet/kubelet_monitor.go

```diff
@@ -172,13 +172,24 @@ func (m *KubeletMonitor) parsePodStats(podStats []stats.PodStats) {
 		pod := &(podStats[i])
 		cpuUsed, memUsed := m.parseContainerStats(pod)
 
+		ephemeralFsCapacity, ephemeralFsUsed := float64(0), float64(0)
+		if pod.EphemeralStorage.CapacityBytes != nil {
+			ephemeralFsCapacity = util.Base2BytesToMegabytes(float64(*pod.EphemeralStorage.CapacityBytes))
+		}
+		if pod.EphemeralStorage.UsedBytes != nil {
+			// OpsMgr server expects the reported size in megabytes
+			ephemeralFsUsed = util.Base2BytesToMegabytes(float64(*pod.EphemeralStorage.UsedBytes))
+		}
+
```

Comment:
Does the `pod.EphemeralStorage.CapacityBytes` represents the sum of the configured limits of ephemeral storage of all containers in a pod (i.e., `spec.containers[].resources.limits.ephemeral-storage`)? What if there is no configured limits (is `CapacityBytes` going to be `nil` in this case)? Shall we then default the limit to the node capacity?

Reply:
@ading1977 I would also have expected what you are listing the behaviour as, however, in my tests, the ephemeral storage capacity of the pod is reported as the node rootfs capacity irrespective of the limits specified on pods containers. Also indicating that it is never nil. 
Let me cross verify the same on other clusters and report back my findings.
If this is indeed the case as to what kubelet reports, I would rather update this PR to get the pod capacity as the sum of limits  specified on the container(s) if the limits exist, and fall back to what is reported as the pod capacity by the kubelet (which is indeed the node rootfs capacity) if the limits are not there on the containers.




You can please hold off merging this PR until then.

Reply:
Findings after further checks.
The resource `capacities` from kubelet are always node capacities. This is true for mem and cpu also. Infact, for all resources kubeturbo reports what kubelet reports and that is node capacity, irrespective of limits specified on the containers.
This is probably because, limits are something a user request's and not really a pods capacity. A pods capacity is only bound by the compute resources provided by the host, ie. the node.

Given above, I think the PR here is good to go.
@ading1977 
cc @enlinxu